web-marketplace/src/components/organisms/LoginModal/components/LoginModal.Mobile.tsx

```diff
@@ -1,5 +1,6 @@
 import { useEffect, useState } from 'react';
```

Comment:
For components that have storybook stories, we get an error: https://deploy-preview-2449--regen-storybook.netlify.app/?path=/story/marketplace-organisms-loginmodal--default
I guess we need to use the i18n provider in those stories?

Reply:
I have added a I18n provider to storybook in APP-201 (still WIP) so we don't have to add the provider to individual stories. The `web-storybook/.storybook/preview.tsx` file has now a fix for the Buffer error and for the I18n stories error in storybook. The file currently looks like this: 
```
import React from 'react';
import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
import { CssBaseline } from '@mui/material';
import { BrowserRouter as Router } from 'react-router-dom';
import { Buffer } from 'buffer';
import { I18nProvider } from '@lingui/react';
import { i18n } from '@lingui/core';

import ThemeProvider from 'web-components/src/theme/RegenThemeProvider';
import 'web-components/src/theme/fonts.css';
import '../../tailwind.css';

window.Buffer = Buffer;
i18n.activate('en');

export const decorators = [
  Story => (
    <I18nProvider i18n={i18n}>
      <ThemeProvider injectFonts>
        <CssBaseline />
        <Router>
          <Story />
        </Router>
      </ThemeProvider>
    </I18nProvider>
  ),
];

export const parameters = {
  viewport: {
    viewports: {
      ...INITIAL_VIEWPORTS,
    },
  },
};
```

Reply:
I made some additional changes to the `preview.jsx` file because there was still an issue with the storybook (translations weren't loaded).
https://github.com/regen-network/regen-web/pull/2449/commits/394e97541b1e29f883b154241977c27e6e4edeab